app/src/lib/git/stash.ts

```diff
@@ -144,23 +144,53 @@ function extractBranchFromMessage(message: string): string | null {
   return match === null || match[1].length === 0 ? null : match[1]
 }
 
-/** Get the files that were changed in the given stash commit.
- *  This is different than `getChangedFiles` because stashes
- *  have _3 parents(!!!)_
+/** The SHA for the null tree. */
```

Comment:
:memo: pedantic feedback

```suggestion
/** The SHA for the empty tree. */
```

I'd link off to an external discussion about what this value represents. This is my favourite currently: https://stackoverflow.com/questions/9765453/is-gits-semi-secret-empty-tree-object-reliable-and-why-is-there-not-a-symbolic